metricbeat/module/elasticsearch/index/index.go

```diff
@@ -114,21 +118,24 @@ func (m *MetricSet) updateServicePath(esVersion common.Version) error {
 
 func getServicePath(esVersion common.Version) (string, error) {
 	currPath := statsPath
-	if esVersion.LessThan(elasticsearch.BulkStatsAvailableVersion) {
-		// Can't request bulk stats so don't change service URI
-		return currPath, nil
-	}
-
 	u, err := url.Parse(currPath)
 	if err != nil {
 		return "", err
 	}
 
-	if strings.HasSuffix(u.Path, ",bulk") {
-		// Bulk stats already being requested so don't change service URI
-		return currPath, nil
+	if !esVersion.LessThan(elasticsearch.BulkStatsAvailableVersion) {
+		if !strings.HasSuffix(u.Path, bulkSuffix) {
+			u.Path += bulkSuffix
+		}
+	}
+
+	if !esVersion.LessThan(elasticsearch.ExpandWildcardsHiddenAvailableVersion) {
+		ew := u.Query().Get(expandWildcardsParam)
+		if !strings.HasSuffix(ew, hiddenSuffix) {
```

Comment:
The path comes from a constant, do we need to do these checks to see if some suffix is already present?

Reply:
Yeah, I guess you're right. I was thinking of a different problem (https://github.com/elastic/beats/pull/16044) but that's not applicable here. I will remove the checks.

Reply:
Removed in 854671e.